Anytype/Sources/PresentationLayer/TextEditor/Assembly/EditorScreenData+Details.swift

```diff
@@ -72,4 +74,15 @@ extension EditorScreenData {
             return object.objectId
         }
     }
+    
+    var spaceId: String? {
+        switch self {
+        case .favorites, .recent, .sets, .collections, .bin:
+            return nil
+        case .page(let object):
+            return object.objectId
+        case .set(let object):
+            return object.objectId
+        }
+    }
```

Comment:
looks strange for me. Is `objectId` right value here?

Reply:
Yes, thanks!